### PR TITLE
boards/arm/samv7: Update mcuboot configs

### PR DESCRIPTION
**Pull request by nandojve**
## Summary

Update MCUboot configs due to https://github.com/apache/incubator-nuttx-apps/pull/929
Add mcuboot-ota example config for samv7-xult board.

## Impact

Any mcuboot board supported.

## Testing

Pass CI
samv7-xult

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
@nandojve is both patch ready for merge?

**Comment by pkarashchenko** [comment]
@xiaoxiang781216 let's rebase to the latest mainline before merge. I will be able to verify before merge

**Comment by xiaoxiang781216** [comment]
> @xiaoxiang781216 let's rebase to the latest mainline before merge. I will be able to verify before merge

Ok, let's wait your verification.

**Comment by nandojve** [comment]
Hi @pkarashchenko , @xiaoxiang781216 ,

I double checked and this is supposed to be OK together with https://github.com/apache/incubator-nuttx-apps/pull/929.

**Review by pkarashchenko — APPROVED** [review]

File: boards/arm/samv7/samv71-xult/README.txt [hunk 1]
```diff
@@ -2639,15 +2639,110 @@ Configuration sub-directories
...
+    *** Booting MCUboot build 7c890f4b075aed73e4c825ccf875b2fb9ebf2ded ***
+    Application Image successfully confirmed!
+
+    Relevant configuration settings:
+
+      CONFIG_EXAMPLES_MCUBOOT_SLOT_CONFIRM=y
 
       CONFIG_SAMV7_FORMAT_MCUBOOT=y
-      CONFIG_INIT_ENTRYPOINT="nsh_main"
+      CONFIG_USER_ENTRYPOINT="nsh_main"
```
Comment (pkarashchenko) [inline boards/arm/samv7/samv71-xult/README.txt:2748]:
```suggestion
      CONFIG_INIT_ENTRYPOINT="nsh_main"
```

File: boards/arm/samv7/samv71-xult/README.txt [hunk 2]
```diff
@@ -2639,15 +2639,110 @@ Configuration sub-directories
...
+    which will be used in the download process.
+
+    See mcuboot-confirm for more information.
+
+    Relevant configuration settings:
+
+      CONFIG_EXAMPLES_MCUBOOT_UPDATE_AGENT=y
+
+      CONFIG_SAMV7_FORMAT_MCUBOOT=y
+      CONFIG_USER_ENTRYPOINT="nsh_main"
```
Comment (pkarashchenko) [inline boards/arm/samv7/samv71-xult/README.txt:2690]:
```suggestion
      CONFIG_INIT_ENTRYPOINT="nsh_main"
```

File: boards/arm/samv7/samv71-xult/README.txt [hunk 3]
```diff
@@ -2639,15 +2639,110 @@ Configuration sub-directories
...
 	      -c 'reset run' -c shutdown
 
     Relevant configuration settings:
 
-      CONFIG_BOOT_MCUBOOT=y
-      CONFIG_MCUBOOT_SLOT_CONFIRM_EXAMPLE=y
+      CONFIG_EXAMPLES_MCUBOOT_SWAP_TEST=y
+
+      CONFIG_SAMV7_FORMAT_MCUBOOT=y
+      CONFIG_USER_ENTRYPOINT="nsh_main"
```
Comment (pkarashchenko) [inline boards/arm/samv7/samv71-xult/README.txt:2652]:
```suggestion
      CONFIG_INIT_ENTRYPOINT="nsh_main"
```

**Review by pkarashchenko — CHANGES_REQUESTED** [review]

**Comment by nandojve** [comment]
Thank you @pkarashchenko , fixed!

**Review by pkarashchenko — APPROVED** [review]

**Comment by pkarashchenko** [comment]
Merging this with CI failure as there is dependency on PR in apps repository.

**Pull request merged**